errors/duplicate.error.js

```diff
@@ -0,0 +1,9 @@
+class DuplicateError extends Error {
```

Comment:
I woudnt use DuplicateError name for 400 status code. It's better to use more common name, f.e. BadRequestError